### ISSUE TITLE
getblocksubsidy returns wrong values post HF

### ISSUE THREAD

**Issue by cronicc** [HorizenOfficial/zen#40]
Current
```
zen-cli getblocksubsidy
{
  "miner": 9.11250000,
  "community": 3.38750000
}
```
Should be 
```
{
  "miner": 8.75000000,
  "community": 3.75000000
}
```

**Comment by cronicc** [comment]
Or better
```
{
  "miner": 8.75000000,
  "community": 1.25000000,
  "securenodes: 1.25000000,
  "supernodes: 1.25000000
}
```

**Comment by pierstab** [comment]
closed by this #48
